apps/builder/app/builder/features/ai/copywriter.tsx

```diff
@@ -134,12 +116,30 @@ export const Copywriter = () => {
                   },
                 ],
                 {
-                  onChunk,
+                  onChunk: (id, { decoded }) => {
+                    try {
+                      const jsonResponse = z
+                        .array(copywriter.TextInstanceSchema)
+                        .parse(JSON.parse(untruncateJson(decoded)));
+
+                      const currenTextInstance = jsonResponse.pop();
+
+                      if (currenTextInstance === undefined) {
+                        return;
+                      }
+
+                      patchTextInstance(currenTextInstance);
+                    } catch {
+                      /**/
+                    }
+                  },
                 }
               ).then((result) => {
                 abort.current = null;
-                if (typeof result !== "string") {
-                  alert("Error " + result.type);
+                if (result.success === false) {
+                  // eslint-disable-next-line no-console
+                  console.log(result);
```

Comment:
Forgotten? If not better use info/warn/error

Reply:
yep, please don't disable that lint rule, we should use log only for debugging so we don't forget to remove